cmd/kubeadm/app/preflight/checks.go

```diff
@@ -829,14 +829,15 @@ func (ImagePullCheck) Name() string {
 // Check pulls images required by kubeadm. This is a mutating check
 func (ipc ImagePullCheck) Check() (warnings, errors []error) {
 	for _, image := range ipc.imageList {
-		glog.V(1).Infoln("pulling ", image)
 		ret, err := ipc.runtime.ImageExists(image)
 		if ret && err == nil {
+			glog.V(1).Infof("image %s exists", image)
 			continue
 		}
 		if err != nil {
 			errors = append(errors, fmt.Errorf("failed to check if image %s exists: %v", image, err))
 		}
+		glog.V(1).Infoln("pulling ", image)
```

Comment:
Infoln -> Infof

Reply:
I am wondering whether we should remove L838, which makes it confusing for users.

Reply:
with this change it will only error on actual errors and not if the image is missing:
https://github.com/kubernetes/kubernetes/pull/66822
@bart0sh to confirm.

Reply:
@neolit123 
> with this change it will only error on actual errors and not if the image is missing:
#66822

Current implementation never returns errors. It just runs 'docker inspect' or 'crictl inspect' and checks its exit code. Any error means that image doesn't exist.

Frankly I think this runtime.ImageExists can be omitted as 'docker pull' and 'crictl pull' always check if image already exists before pulling. We're just duplicating this functionality.